purchase_stock_analytic/models/purchase_order_line.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2020 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class PurchaseOrderLine(models.Model):
+
+    _inherit = "purchase.order.line"
+
+    def _prepare_stock_moves(self, picking):
+        res = super(PurchaseOrderLine, self)._prepare_stock_moves(picking)
+        for line in self:
+            aa = line.account_analytic_id
+            at = line.analytic_tag_ids
```

Comment:
Can we please improve the variable names lie aa to analytic_account and at to analytic_tags ?